### PR TITLE
feat: new style for settings

### PR DESCRIPTION
**Pull request by jleach**
# Summary of Changes

Style the settings screen as per the designs.

# Related Issues

Please reference here any issue #'s that are relevant to this PR, or simply enter "N/A" if this PR does not relate to any existing issues.

# Pull Request Checklist

Tick all boxes below to demonstrate that you have completed the respective task. If the item does not apply to your this PR **check it anyway** to make it apparent that there's nothing to do.

- [x] All commits contain a DCO `Signed-off-by` line (we use the [DCO GitHub app](https://github.com/apps/dco) to enforce this);
- [x] Updated LICENSE-3RD-PARTY.md for any added dependencies or vendored components;
- [x] Updated documentation as needed for changed code and new or modified features;
- [x] Added sufficient [tests](../__tests__/) so that overall code coverage is not reduced.

If you have _any_ questions to _any_ of the points above, just **submit and ask**! This checklist is here to _help_ you, not to deter you from contributing!

Pro Tip 🤓

- Read our [contribution guide](../CONTRIBUTING.md) at least once; it will save you a few review cycles!
- Your PR will likely not be reviewed until all the above boxes are checked and all automated tests have passed.

_PR template adapted from the Python attrs project._

### CODE REVIEW

File: core/App/screens/Settings.tsx [hunk 1]
```diff
@@ -6,80 +6,149 @@ import { getVersion, getBuildNumber } from 'react-native-device-info'
...
+                  ?.navigate(Stacks.ContactStack, { screen: Screens.Contacts, params: { navigation: navigation } })
+              }
+            />
+            {/* <SeparatorLine />
+            <Row
+              title={'What are Contacts?'}
+              accessibilityLabel={'What are Contacts?'}
+              testID={testIdWithKey('WhatContacts')}
+              onPress={() => null}
+            /> */}
```
Comment (wadeking98) [inline core/App/screens/Settings.tsx:114]: should we get rid of the commented out section?

File: core/App/screens/Settings.tsx [hunk 2]
```diff
@@ -6,80 +6,149 @@ import { getVersion, getBuildNumber } from 'react-native-device-info'
...
+              onPress={() => null}
+            />
+            <SeparatorLine />
+            {/* <Row
+              title={'Change PIN'}
+              accessibilityLabel={'Change PIN'}
+              testID={testIdWithKey('ChangePIN')}
+              onPress={() => null}
+            />
+            <SeparatorLine /> */}
```
Comment (wadeking98) [inline core/App/screens/Settings.tsx:135]: Commented out section here too

File: core/App/screens/Settings.tsx (hunk 1 above)
Comment (jleach) [reply core/App/screens/Settings.tsx:114]: I'm going to use it in future PRs to follow this one. Will get Ontarios work in first then add these back int.

**Review by wadeking98 — APPROVED** [review]
Couple of comments regarding commented out lines but it looks good to merge aside from that

File: core/App/screens/Settings.tsx (hunk 2 above)
Comment (jleach) [reply core/App/screens/Settings.tsx:135]: Same as above. Will be used in the next PRs.

**Review by wadeking98 — APPROVED** [review]

**Review by cvarjao — APPROVED** [review]

**Pull request merged**